### PR TITLE
Adds request and bin controllers to main app file

### PR DESCRIPTION
**Pull request by gcpinckert**
This won't work as of yet, as neither controller is working, but I'd like to merge these changes with main so that I can test pulling an URL param from the "root path" passed to `router.use` on line 18 of `app.js` in `controllers/requests.js`.

Thanks!

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -7,10 +7,18 @@ require("./db/pg.js");
 
 // Logging
 const morgan = require("morgan");
+const router = require("./controllers/collect");
```
Comment (lukeoguro) [inline app.js]: I don't think we need this line

File: app.js [hunk 2]
```diff
@@ -7,10 +7,18 @@ require("./db/pg.js");
...
 app.use(morgan("tiny"));
 
 // Collect API
 const collectRouter = require('./controllers/collect');
 app.use('/collect', collectRouter)
 
+// Requests and Bins APIs
+const requestController = require('./controllers/collect');
+router.use('/api/bins/:uuid/requests', requestController)
```
Comment (lukeoguro) [inline app.js]: I think this is `app.use`

File: app.js [hunk 3]
```diff
@@ -7,10 +7,18 @@ require("./db/pg.js");
...
+
+const binController = require('./controllers/binController');
+router.use('/api/bins', binController)
```
Comment (lukeoguro) [inline app.js]: Same, `app.use`

**Review by lukeoguro — CHANGES_REQUESTED** [review]
Left some comments, lemme know if you have questions

**Comment by gcpinckert** [comment]
Yeah I saw those :woman-facepalming:, just pushed commit that should fix the typos

File: app.js (hunk 1 above)
Comment (gcpinckert) [reply app.js]: oh weird I don't know where that came from.
Will push one more commit

**Review by lukeoguro — APPROVED** [review]
looks good to me!

**Review by westrachel — APPROVED** [review]
these updates look good

**Pull request merged**